packages/tools/examples/stackAnnotationTools/index.ts

```diff
@@ -180,9 +181,14 @@ addButtonToToolbar({
   },
 });
 
-addButtonToToolbar({
-  title: 'Rotate Delta 90',
-  onClick: () => {
+addSliderToToolbar({
```

Comment:
I know you used this example for debugging, but I rather keep this one simple so can you take this out please?

Reply:
Thanks for the suggestion! I've completed the requested tasks. Should I create new `/tools/examples/.../index.ts` example where it is possible to add ROI tools and enable rotation or it's ok for now?